cchecker.py

```diff
@@ -18,23 +17,21 @@ class SupportedTests(argparse.Action):
 A custom argparse Action to ensure only the supported tests are passed in
         """
         def __call__(self, parser, namespace, values, option_string=None):
```

Comment:
is this custom handler needed anymore rather than the built in `nargs='+'` / `choices=` behavior?